archive/archive_ingest/src/archive_ingest.py

```diff
@@ -0,0 +1,68 @@
+# -*- encoding: utf-8 -*-
+"""
+Receives a message to ingest a bag giving the URL and publishes the archive event to an SNS topic.
+"""
+
+import os
+
+import boto3
+import daiquiri
+from urllib.parse import urlparse
```

Comment:
style: convention is that Python imports go in three groups – builtins, third-party modules, first-party code. Since `urllib` is a standard library module, better would be:

```python
import os
from urllib.parse import urlparse

import boto3
import daiquiri
```